client/apps/edit/components/content2/sections/header/index.jsx

```diff
@@ -22,24 +23,13 @@ export default class SectionHeader extends Component {
     const heroSection = this.props.article.heroSection
     heroSection.set(key, value)
     this.onChange('hero_section', heroSection.attributes)
-  }
-
-  onChangeLeadParagraph = (html) => {
-    this.onChange('lead_paragraph', html)
-  }
-
-  onUpload = (image, width, height) => {
-    this.onChangeHero('url', image)
+    this.forceUpdate() // re-render reaction header
```

Comment:
The [React docs](https://reactjs.org/docs/react-component.html#forceupdate) suggest to avoid using `forceUpdate` when you can as it skips lifecycle methods (which further down the line is weird to debug). A better way to trigger updates in this fashion would be to use `setState`:
```js
  onChangeHero = (key, value) => {
    ...
    this.setState({
      refreshedAt: Date.now()
    })
  }
```